samples/bluetooth/peripheral_uart/child_image/hci_rpmsg/boards/thingy53_nrf5340_cpunet.conf

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_RPMSG_SERVICE=y
+CONFIG_RPMSG_SERVICE_MODE_REMOTE=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+
+CONFIG_MAIN_STACK_SIZE=512
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_MAX_CONN=10
+CONFIG_BT_CTLR_ASSERT_HANDLER=y
+CONFIG_BT_HCI_RAW_RESERVE=1
+CONFIG_BT_BUF_CMD_TX_COUNT=10
```

Comment:
What's the reason for having to repeat configurations? Aren't the applied properly as config fragments?

Reply:
We need to define board-specific child image configuration. Since I did not want to modify hci_rpmsg configuration for other supported boards I defined only (E) without defining (G). In that case whole configuration is placed in (E).

See docs: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html for explanation of used letters.

Reply:
That's unfortunate, the workaround for the command buffers is now spread out. Have you checked if there are configurations that will now have this one as missing?

Could you add a reference to this in NCSDK-10851 so that we can reverse the workaround applied here once it is resolved?

Reply:
Added reference to Thingy:53 under the mentioned Jira ticket.

Reply:
The fix is included in all of my configurations, so the workaround should work fine :)